src/QueryBuilder/ResultViewer/ResultViewer.js

```diff
@@ -191,6 +191,7 @@ export const ResultViewer = ({
   showQueryAccordion,
   fqlQuery,
   height,
+  autosize,
```

Comment:
Could you please clarify if this is not a required field, should a default value be added?